app/src/main/res/values/strings.xml

```diff
@@ -179,6 +179,14 @@
     <!-- Preference switch for app health report. The first parameter is the name of the application (For example: Fenix) -->
     <string name="preferences_fenix_health_report">%s health report</string>
 
+    <!-- Turn On Sync Preferences -->
+    <!-- Header of the Turn on Sync preference view -->
+    <string name="preferences_sync">Turn on Sync</string>
```

Comment:
do I need to add these strings somewhere else?

Reply:
Nope! This is good